sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Constraints/BallSocketConstraintComponent.cs

```diff
@@ -44,6 +66,10 @@ public Vector3 LocalOffsetB
         }
     }
 
+    /// <summary>
+    /// Gets or sets the target number of undamped oscillations per unit of time.
+    /// Higher frequency values create stiffer connections, while lower values allow more elasticity in the joint.
+    /// </summary>
```

Comment:
@Eideren, updated, could you review this again? If it's ok I would add it to all other constraints in this PR.

Reply:
Ok. I will let you know once this PR is ready.